std_msgs/String file_name
geometry_msgs/Pose pose